mtags-java/src/main/scala/scala/meta/internal/pc/JavaCompletionProvider.scala

```diff
@@ -51,20 +51,15 @@ class JavaCompletionProvider(
 
     node match {
       case Some(n) =>
-        val list =
-          n.getLeaf.getKind match {
-            case MEMBER_SELECT => completeMemberSelect(task, n)
-            case IDENTIFIER => completeIdentifier(task, n)
-            case _ => new CompletionList()
-          }
-
-        val keywordsCompletion = keywords(n)
-        val resultList =
-          (list.getItems.asScala ++ keywordsCompletion.getItems.asScala).distinct
-            .sorted(ordering)
-            .asJava
-
-        new CompletionList(resultList)
+        val items = n.getLeaf.getKind match {
+          case MEMBER_SELECT => completeMemberSelect(task, n)
+          case IDENTIFIER =>
+            val scopeItems = completeIdentifier(task, n)
+            val keywordItems = keywords(n)
+            (scopeItems ++ keywordItems).distinct.sorted(ordering)
+          case _ => keywords(n)
+        }
+        new CompletionList(items.asJava)
```

Comment:
```suggestion
            (scopeItems ++ keywordItems)
          case _ => keywords(n)
        }
        new CompletionList(items.distinct.sorted(ordering).asJava)
```
Don't we want to sort in every case?

Reply:
Good idea, changed!